examples/upload-media-from-local/README.md

```diff
@@ -0,0 +1,41 @@
+# Upload media from local
+
+This is a script to handle uploading media locally to Studio.
+
+---
+
+## How to use
+
+For using this script we should run it through runner script in `bin` folder:
+
+```bash
+$ bin/run examples/upload-media-from-local/upload.py <arguments>
+```
+
+here is the list of arguments we can use:
```

Comment:
I believe this is too verbose, I'd rather have three version of the sample CLI for each case ( single upload, multiple uplod and one of them with the user id parameter)